src/runtime_impl.cpp

```diff
@@ -60,9 +60,6 @@ class runtime_impl : public runtime {
             RAC_ERRNO_MSG("runtime_impl before creating the dynamic library");
             return dynamic_lib::create(so_path);
         });
-        for (auto& i : libraries_) {
```

Comment:
`enable_monado` is defaulted to `false` now, and it will not work for the server side where the display is provided by monado.